### ISSUE TITLE
Image in pptx do not work

### ISSUE THREAD

**Issue by clodio** [open-xml-templating/docxtemplater-image-module#72]
I tried to use a pptx instead of a docx, but i got an error
```
C:\node\testDocxtemplater\node_modules\docxtemplater\js\traits.js:15
        throw err;
        ^
Error: Raw tag not in paragraph
    at Error.XTTemplateError (C:\node\testDocxtemplater\node_modules\docxtemplater\js\errors.js:13:15)
    at throwRawTagNotInParagraph (C:\node\testDocxtemplater\node_modules\docxtemplater\js\traits.js:7:12)
    at expandOne (C:\node\testDocxtemplater\node_modules\docxtemplater\js\traits.js:84:3)
    at C:\node\testDocxtemplater\node_modules\docxtemplater\js\traits.js:105:16
    at Array.forEach (native)
    at Object.expandToOne (C:\node\testDocxtemplater\node_modules\docxtemplater\js\traits.js:104:19)
    at ImageModule.postparse (C:\node\testDocxtemplater\node_modules\docxtemplater-image-module\js\index.js:74:27)
    at C:\node\testDocxtemplater\node_modules\docxtemplater\js\parser.js:14:38
    at Array.reduce (native)
    at postparse (C:\node\testDocxtemplater\node_modules\docxtemplater\js\parser.js:13:19)

```
It does not seem to be like #11 

My code is availabable on github : https://github.com/clodio/testDocxtemplater

or underneath

```
var fs = require('fs');
var Docxtemplater = require('docxtemplater');
var JSZip = require('jszip');
var ImageModule=require('docxtemplater-image-module')

var content = fs.readFileSync(__dirname+"/imageExample.pptx","binary");

var opts = {
  getImage: function (tagValue) {
    return fs.readFileSync(tagValue, "binary");
  },
  getSize: function () {
    return [150, 150];
  },
  centered: false,
};

var imageModule=new ImageModule(opts);

var zip = new JSZip(content);
var docx=new Docxtemplater()
    .loadZip(zip)
    .setData({ image: 'examples/image.png' })
    .setOptions({fileType:'pptx'})
    .attachModule(imageModule)
    .render();

var buffer= docx
        .getZip()
        .generate({type:"nodebuffer"});
        console.log(docx)

fs.writeFile("imageExample_output.pptx",buffer);
```

**Comment by edi9999** [comment]
It is the same issue as https://github.com/open-xml-templating/docxtemplater-image-module/issues/11 in the end.

https://github.com/open-xml-templating/docxtemplater-image-module/issues/11 was opened in version 1.x and the pull request was never merged because of code quality. Now that we are in version 3.0, the code still doesn't handle pptx, because I don't have much time to do it. The error has changed since then, but someone could open a new pull request by having a look at the proposed patch at : https://github.com/open-xml-templating/docxtemplater-image-module/pull/12

**Comment by edi9999** [comment]
tldr : pptx is not handled by this module yet, but this is potentially possible

**Comment by edi9999** [comment]
https://github.com/open-xml-templating/docxtemplater-image-module/pull/74

it is now possible !nn
